crates/bevy_pbr/src/meshlet/meshlet_bindings.wgsl

```diff
@@ -7,23 +7,19 @@
 struct PackedMeshletVertex {
     a: vec4<f32>,
     b: vec4<f32>,
-    tangent: vec4<f32>,
 }
 
-// TODO: Octahedral encode normal, remove tangent and derive from UV derivatives
 struct MeshletVertex {
     position: vec3<f32>,
     normal: vec3<f32>,
```

Comment:
Could still use oct for the normal

Reply:
Yeah the plan is still to do that, but in a future PR. I'm going to be switching to a per-meshlet vertex buffer (still packed into one large array ofc), and quantize and compress the vertex postions. When I do that, I'll also be switching to octahedral normals.

I didn't do it now both to try and make smaller PRs, and because it would throw alignment off and need a bunch more work to pack things properly when I'm just going to need to redo it later anyways. Removing the tangents is an easy win and doesn't mess with the alignment.